PokeDemo/Controllers/HomeViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ViewController.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+// MARK: - Navigation
+
+import UIKit
+
+class HomeViewController: BaseViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var pokemonTableView: UITableView!
+    
+    // MARK: - Properties
+    
+    // MARK: - Private
+
+    private let presenter = HomeViewPresenter(service: Service.shared)
```

Comment:
default shared parameter can be passed.

```
init(service: Service = Service.shared)
```
and thus can be used simply as - 
```
HomeViewPresenter()
```

Reply:
Done in commit cd97b93468f78a9f35b47f3476ee786dce2aca8a

Reply:
Next time create commit only with changes mentioned in the comment. Your mentioned commit doing so many unrelated changes which is not part of this comment